apps/extensions/src/content/panel/screens/favoriteScreen/Favorites.tsx

```
@@ -0,0 +1,239 @@
+                  <img
+                    src={noImage}
+                    style={{ width: '100px', height: '100px' }}
+                    alt="no-image"
+                  />
+                  <h1 className="tw-text-xl tw-font-bold tw-text-center">
+                    No Favourites Images Folders found
+                  </h1>
+                </div>
+              </div>
+            )}
+            <div className={styles.pageHeadingWrapper}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)